cms/templates/js/no-group-configurations.underscore

```diff
@@ -1,3 +1,3 @@
 <div class="no-group-configurations-content">
-    <p><%= gettext("You haven't created any group configurations yet.") %><a href="#" class="button new-button"><i class="icon-plus"></i><%= gettext("Add your first Group Configuration") %></a></p>
+    <p><%= gettext("You haven't created any group configurations yet.") %><a href="#" class="button new-button"><i class="icon-plus"></i><%= gettext("Add your first Experiment Configuration") %></a></p>
```

Comment:
@talbs I don't necessarily agree that the Grammar Force is stronger with me than with you! I do agree that we can go without caps mid-sentence; just wanted to check with you about how other instances of "Add your first..." button text are handled. Sounds good to me. Thanks!